src/lib/reasoners/fun_sat.ml

```diff
@@ -1637,13 +1632,6 @@ module Make (Th : Theory.S) = struct
       ~init:env
 
   let pop env to_pop =
-    if Options.get_tableaux_cdcl () then
-      Errors.run_error
-        (Errors.Unsupported_feature
-           "Incremental commands are not implemented in \
-            Tableaux(CDCL) solver ! \
-            Please use the Tableaux or CDLC SAT solvers instead"
-        );
```

Comment:
I thought this was still not working — just commenting here to make sure we don't merge until either this works with Tableaux(CDCL) (probably not worth it, waiting on @Halbaroth investigation whether we should remove Tableaux(CDCL)) or to keep the assert back in.